### PR TITLE
Crud operations book

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Completed crud endpoints of Book entity with custom exceptions

## Trello Card
https://trello.com/c/7M74pv21/18-crear-las-operaciones-crud-de-book

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -1,18 +1,70 @@
...
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import wolox.training.exceptions.BookIdMismatchException;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
 @Controller
+@RequestMapping(path = "/books")
+
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/controllers/BookController.java:24]: Hello! I'll check the card but I think we need a getByAuthor path 😁

**Review by julianvasquez-wolox — DISMISSED** [review]
Good job! 😁

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,8 +1,8 @@
-
 spring.datasource.url=jdbc:postgresql://localhost:5432/mytest
-spring.datasource.username= postgres
-spring.datasource.password= user
-spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto = update
-server.port = 8081
+spring.datasource.username=postgres
+spring.datasource.password=user
```
Comment (Jimenarosello) [inline src/main/resources/application.properties]: Just as a comment (no need to change anything 😅) remember that in general the credentials and sensitive info are set as environment variables/secrets so as to not push them to the repo :) for the training purposes this is fine, so don't worry 😄

**Review by Jimenarosello — DISMISSED** [review]
Nice job!!

**Review by julianvasquez-wolox — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
Great! 😄

**Pull request merged**